packages/@vue/cli-ui/src/components/ListItemInfo.vue

```diff
@@ -12,7 +12,7 @@
     </div>
     <div v-if="description || link || showDescription" class="description">
       <slot name="description">
-        <span v-html="description"/>
```

Comment:
Why this change?

Reply:
Description of some rules contain `<tagname>` and it broked the UI 🤔

Reply:
Could we strip them out?

Reply:
escaped